tests/integration-tests/tests/storage/test_raid.py

```diff
@@ -75,7 +75,7 @@ def _test_raid_correctly_configured(remote_command_executor, raid_type, volume_s
     assert_that(result.stdout).matches(r"Array Size : .*{0}\.00 GiB".format(volume_size))
     assert_that(result.stdout).contains("Raid Devices : {0}".format(raid_devices))
     assert_that(result.stdout).contains("Active Devices : {0}".format(raid_devices))
-    assert_that(result.stdout).contains("Failed Devices : 0".format(raid_devices))
+    assert_that(result.stdout).contains("Failed Devices : {0}".format(raid_devices))
```

Comment:
The correct value here was 0. We only had to remove the format.

Reply:
I see. My bad.